x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/settings/settings.tsx

```diff
@@ -135,6 +109,15 @@ export const SettingsPage: React.FC<Props> = memo(
       kuery: `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name:${name}`,
     });
 
+    const {
+      changelog,
+      isLoading: isChangelogLoading,
+      error: changelogError,
+    } = useChangelog(name, latestVersion, version);
+
+    const breakingChanges = getBreakingChanges(changelog);
+    const hasBreakingChanges = breakingChanges.length > 0;
```

Comment:
nit: probably worth having in a useMemo and may worth moving this to useChangelog to remove some logic from here